README.md

```diff
@@ -6,129 +6,207 @@
-* TheOcean (`theocean-market-maker-keeper`),
-* gate.io (`gateio-market-maker-keeper`).
-
-All these keepers share some logic and operate in a similar way. They create
-a series of orders in so called _bands_, which are configured with a JSON file
-containing parameters like spreads, maximum engagement etc. Please see the
-_"Bands configuration"_ section below for more details regarding keeper mechanics.
-
-Provided an appropriate price feed is available, most of the market maker keepers
-are capable of market-making on any token pair. The EtherDelta and IDEX keepers still are
-to some extend bound to the DAI/ETH pair. This will either be changed at some point in the future,
-or these two keepers will be discontinued.
-
-This repo also contains an auxiliary tool called `oasis-market-maker-cancel`, which
-may be used for emergency cancelling all market maker orders on OasisDEX if the
-keeper gets stuck or dies for some reason, or if the network becomes congested.
-
-<https://chat.makerdao.com/channel/keeper>
-
-
-## Installation
-
-This project uses *Python 3.6.6* and requires *virtualenv* to be installed.
-
-In order to clone the project and install required third-party packages please execute:
+# Market Maker Keeper Setup Guide 
+
+**Guide Outline**
+1. Introduction 
+2. Prerequisites 
+3. Installation 
+4. Testing 
+5. Bands and Bands Configuration
+    - Example
+6. Order Rate Limitation
+7. Data Templating Language
+8. Price Feed Configuration
+    - Example 
+9. Running Keepers
+    - Example (OasisDEX) 
+10. Known limitations
+11. Support Information
+12. License
+
+## 1. Introduction
+A big part of the DAI Stablecoin System (DSS) is the incentivization of external agents, called **Keepers** (which can be human but are typically automated bots). Market Maker Keepers work by creating a series of orders in so-called **bands** (defined later), which are configured with a JSON file containing parameters like spreads, maximum engagement, etc. In short, the `market-maker-keeper` repository is a set of Keepers that facilitate market making on exchanges. For example, trading Dai motivated by the expected long-term convergence toward the indicated `Target Price`. This guide is dedicated to showing you how to create your very own Market Maker Keeper Bot as well as educate the community and help both users and developers understand the value of this incredible software. We are proud to say that all of the code needed to get a Market Maker Keeper bot up and running is open-sourced.
+
+### List of current exchanges that Market Maker Keeper Bots can be built for
+
+- OasisDEX (`oasis-market-maker-keeper`)
+- EtherDelta (`etherdelta-market-maker-keeper`)
+- RadarRelay and ERCdEX (`0x-market-maker-keeper`)
+- Paradex (`paradex-market-maker-keeper`)
+- DDEX (`ddex-market-maker-keeper`)
+- Ethfinex (`ethfinex-market-maker-keeper`)
+- GoPax (`gopax-market-maker-keeper`)
+- OKEX (`okex-market-maker-keeper`)
+- TheOcean (`theocean-market-maker-keeper`)
```

Comment:
Let's add also following keepers to the list:
- 0x
- kraken
- coinbase
- bittrex
- kucoin
- liquid